common/persistence/persistence-tests/executionManagerTest.go

```diff
@@ -1326,7 +1324,8 @@ func (s *ExecutionManagerSuite) TestUpdateWorkflow() {
 	failedUpdateState := copyWorkflowExecutionState(updatedState)
 	err4 := s.UpdateWorkflowExecution(failedUpdateInfo, failedUpdateState, state0.NextEventId, []int64{int64(5)}, nil, int64(3), nil, nil, nil, nil, nil)
 	s.Error(err4, "expected non nil error.")
-	s.IsType(&p.ConditionFailedError{}, err4)
+	fmt.Printf("#######%v#######\n", err4)
```

Comment:
leftover

Reply:
PR not ready for review, but good catch

Reply:
Oh sorry, I thought it was ready for review. Can you put a WIP tag on it?

Reply:
on the left hand side there is an icon indicating whether a PR is a draft or not

![image](https://user-images.githubusercontent.com/8762893/127385584-8e74f5b0-a307-4803-af3d-d2415441556a.png)